documentation/onboard-dpg-in-sdkautomation/add-autorest-configuration-in-spec-comment.md

```diff
@@ -0,0 +1,16 @@
+# Add Autorest Configuration in Spec Comment
+
+## Prerequisites
+There should be a basic readme.md together with swagger, and you can refer to the [sample](../samplefiles-dp).
+
+## Add Autorest Configuration for .Net SDK
+//TODO
```

Comment:
I assume you will fill this in once we agree on the approach and figure out what data is needed for each sdk.

Reply:
added in latest commit